native/src/components/TransparentHeader.tsx

```diff
@@ -76,9 +76,12 @@ class TransparentHeader extends React.PureComponent<PropsType> {
           <HorizontalLeft>
             <HeaderBackButton onPress={this.goBackInStack} labelVisible={false} />
           </HorizontalLeft>
-          <MaterialHeaderButtons cancelLabel={t('cancel')} theme={theme}>
-            {shareUrl && <Item title={t('share')} show='never' onPress={this.onShare} />}
-          </MaterialHeaderButtons>
+          <MaterialHeaderButtons
+            cancelLabel={t('cancel')}
+            theme={theme}
+            items={[]}
+            overflowItems={shareUrl ? [<HiddenItem key='share' title={t('share')} onPress={this.onShare} />] : []}
+          />
```

Comment:
Forgot this when upgrading `react-navigation-header-buttons` since no ts checks where executed in the CI. Should be fixed now.